widgets/common/src/deploymentsView/DeploymentsView.tsx

```diff
@@ -174,6 +174,7 @@ export const DeploymentsView: FunctionComponent<DeploymentsViewProps> = ({
 
     const mapOpen = !!(toolboxContext.getValue(mapOpenContextKey) as boolean | undefined);
     const toggleMap = () => toolboxContext.setValue(mapOpenContextKey, !mapOpen);
+    const deployOnEnabled = !!deploymentsResult.data.items.length;
```

Comment:
This flag is being used only for disabling bulk actions, so WDYT about renaming it to be `disableBulkActions`?

Name `deployOnEnabled` may suggest relationship with different kind of functionalities, which is not the case in this example 😉 

```suggestion
    const deployOnEnabled = !deploymentsResult.data.items.length;
```

(if you'd like to apply the suggestion, other parts of the code should be updated aswell, to use new prop name)

Reply:
good suggestion, added it here: https://github.com/cloudify-cosmo/cloudify-stage/pull/2132/commits/cc2f53dfe9ddb5fccd183f5464a7eaaead52abd0

Reply:
Good idea 👍🏻